src/OrbitQt/CaptureOptionsDialog.h

```diff
@@ -78,6 +81,13 @@ class CaptureOptionsDialog : public QDialog {
   void SetMemoryWarningThresholdKb(uint64_t memory_warning_threshold_kb);
   [[nodiscard]] uint64_t GetMemoryWarningThresholdKb() const;
 
+  // Max to pass to perf_event_open without getting an error is (1u << 16u) - 8,
+  // because the kernel stores this in a short and because of alignment reasons.
+  // But the size the kernel actually returns is smaller and we leave some extra room (see
+  // `PerfEventOpen.cpp`).
+  static constexpr uint16_t kMaxCopyRawStackSizeMaxValue = 65000;
+  static constexpr uint16_t kMaxCopyRawStackSizeDefaultValue = 512;
```

Comment:
I'm not sure how this is dealt with when you need them in two places, but the other defaults are defined in `orbitmainwindow.cpp`, so these are out of place. Either these should be moved there, or the existing ones should be moved here if it's possible.

Reply:
Moved all constants. This also allowed me to remove redefinitions of them in CaptureOptionsDialog.